EIPS/eip-7687.md

```diff
@@ -0,0 +1,144 @@
+---
+eip: 7687
```

Comment:
```suggestion
eip: 7697
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

Reply:
Thanks, I have fixed it.